packages/suite/src/services/coinjoin/config.ts

```diff
@@ -151,6 +151,12 @@ export const CLIENT_STATUS_FALLBACK = {
     },
 };
 
+// Estimating anonymity gain per round:
```

Comment:
I think it would be better to use different word than `hindsight` then it is not required to comment it much.

```
export const ANONYMITY_GAINS_NUM_PREVIOUS_TXS = 10;
export const ANONYMITY_GAINS_OLDEST_TXS_DAYS = 30;
```